chain/chain/Cargo.toml

```diff
@@ -10,7 +10,7 @@ log = "0.4"
 failure = "0.1"
 failure_derive = "0.1"
 lazy_static = "1.4"
-rocksdb = "0.14"
+rocksdb = { git = "https://github.com/nearprotocol/rust-rocksdb", branch="disable-thread" }
```

Comment:
why?

Reply:
It use https://github.com/facebook/rocksdb/pull/7191 (merged, but rust-rocksdb haven't bump version) and https://github.com/rust-rocksdb/rust-rocksdb/pull/448 (approved, but need to wait next time rust-rocksdb bump version). Our fork is equal to https://github.com/rust-rocksdb/rust-rocksdb/pull/448